plugins/toolbox/src/components/Validators/RegexValidator.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react';
+import { TripleEditor } from '../DefaultEditor/TripleEditor';
+
+export const RegexValidator = () => {
+  const [input, setInput] = React.useState('');
+  const [pattern, setPattern] = React.useState('');
+  const [output, setOutput] = React.useState('');
+
+  function isRegexValid(inputString: string, patternString: string): string {
+    const flags = 'gim' // g=global, m=match line start/end, i=case insensitive
+    const regex = new RegExp(patternString, flags);
+    const testResults = regex.exec(inputString);
+    if (testResults === null) {
+      return "pattern doesn't match the input text";
```

Comment:
Please use translations for all messages to user

Reply:
translated